pyanaconda/ui/gui/spokes/subscription.py

```diff
@@ -871,8 +871,7 @@ def _register(self):
 
         # check if the current installation source will be overriden
         # and remember it if it is the case
-        source_proxy = self.payload.get_source_proxy()
-        source_type = source_proxy.Type
+        source_type = self.payload.type
```

Comment:
It should be `self.payload.source_type`.

Reply:
Fixed, thanks! :)